inst/extdata/countrycodes.csv

```diff
@@ -1,4 +1,4 @@
-Code
+code
```

Comment:
btw do these column name changes qualify as breaking changes?

Reply:
Just to be on the safe side?! I would not expect that change to actually break anything though. Do you want me to put that in the "minor changes" category?

Reply:
That'd be great!